coordinator/core/src/test/kotlin/net/consensys/zkevm/ethereum/coordination/aggregation/ProofAggregationCoordinatorServiceTest.kt

```diff
@@ -93,7 +93,7 @@ class ProofAggregationCoordinatorServiceTest {
       consecutiveProvenBlobsProvider = mockAggregationsRepository::findConsecutiveProvenBlobs,
       proofAggregationClient = mockProofAggregationClient,
       aggregationL2StateProvider = aggregationL2StateProvider,
-      metricsFacade = MicrometerMetricsFacade(registry = SimpleMeterRegistry()),
+      metricsFacade = mock<MetricsFacade>(defaultAnswer = Mockito.RETURNS_DEEP_STUBS),
```

Comment:
What is your motivation to use a mock instead of the real implementation? 

I encourage you to read [this section](https://abseil.io/resources/swe-book/html/ch13.html#real_implementations) about favoring Real Implementation of mocks